modules/ROOT/pages/provisioning-kubevirt.adoc

```diff
@@ -0,0 +1,89 @@
+= Provisioning Fedora CoreOS on KubeVirt
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on any KubeVirt-enabled Kubernetes cluster.
+
+== Prerequisites
+
+Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to a Kubernetes environment with https://kubevirt.io/user-guide/operations/installation/[KubeVirt] installed.
+
+== Referencing the KubeVirt Image
+
+Fedora CoreOS is designed to be updated automatically, with different schedules per stream.
+
+The image for each stream can directly be referenced from the official registry:
+
+----
+quay.io/fedora/fedora-coreos-kubevirt
+----
+
+== Launching a VM instance
+
+Now that you have an image it can be used in your VMI definitions. The following example will use the image from the stable stream.
+
+You'll also need the Ignition config.
```

Comment:
Let's drop the reference to the stream here (we'll select that below) in the "code" and also combine a few sentences:

```suggestion
Given the `quay.io/fedora/fedora-coreos-kubevirt` images you can create a VMI defnition and combine that with an Ignition config to launch a machine.

In the example below...
```